<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>220</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>220</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>260</y>
      <width>220</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>260</y>
      <width>220</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Стандартные номера:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>137</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Комфорт номера:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>123</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Бизнес номера:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>220</y>
      <width>109</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Люкс номера:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddRoom">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>190</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Настройки этажа
           1</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images for project/icon_settings.png</normaloff>../images for project/icon_settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenAdmin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>144</width>
      <height>78</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Войти в
систему
как админ</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images for project/icon_admin.png</normaloff>../images for project/icon_admin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetRooms">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>144</width>
      <height>78</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Сбросить
 все 
комнаты</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images for project/icon_reset.png</normaloff>../images for project/icon_reset.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenListGuests">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>144</width>
      <height>78</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Открыть
список
гостей</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images for project/image_MainWindow.png</normaloff>../images for project/image_MainWindow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitAdmin">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>20</y>
      <width>144</width>
      <height>78</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выйти
 из режима
 админа</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images for project/icon_admin.png</normaloff>../images for project/icon_admin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AddRoom_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>190</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Настройки этажа
           2</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images for project/icon_settings.png</normaloff>../images for project/icon_settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="AddRoom_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>160</y>
      <width>190</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Настройки этажа
           3</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images for project/icon_settings.png</normaloff>../images for project/icon_settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="AddRoom_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>160</y>
      <width>190</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Настройки этажа
           4</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images for project/icon_settings.png</normaloff>../images for project/icon_settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="FoundFastRoom">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>151</width>
      <height>78</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Найти
ближайшую
комнату</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images for project/icon_room.png</normaloff>../images for project/icon_room.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="DoReport">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>151</width>
      <height>78</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Сделать
отчет</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images for project/report_icon.png</normaloff>../images for project/report_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1130</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
